### ISSUE TITLE
Top-right cell only sends Loudness/Z of either 0 or 127

### ISSUE THREAD

**Issue by martinfinke** [rogerlinndesign/linnstrument-firmware#41]
Since commit a128c565eed7c6af46aa6be818d4ea5649c6ded9, the cell in the very top-right corner on my Linnstrument only sends Loudness/Z values of either 0 or 127. Velocity and Timbre/Y work fine. Aftertouch on the three other corner cells still work fine. I've looked at the commit but couldn't quite figure out what caused this..

Not a big problem because I'm fine with v2.2.1 where it's working fine 🙂But thought it might be good to report this.

**Comment by gbevin** [comment]
Are you absolutely sure it's since that exact commit?

**Comment by martinfinke** [comment]
Yeah, on my test setup it stopped working on that exact commit. I cherry-picked each of the 7 commits on master since the `2.2.1` tag. After each cherry-pick I built and uploaded the firmware, and checked the MIDI messages from pressing the cell. It worked with all commits before that one, and stopped working after I cherry-picked/built/uploaded a128c565eed7c6af46aa6be818d4ea5649c6ded9.
Hope this helps 😊
